source/default/tailwind.config.js

```diff
@@ -3,85 +3,106 @@
  *   Requires Tailwind 1.4.0 or greater
  */
 
-const path = require('path');
+ const path = require('path');
```

Comment:
Looks like most of changes here are due to extra indent space. Not a big deal, but sure makes change review more difficult.